pym/bob/cmds/build.py

```diff
@@ -412,6 +412,7 @@ def _runShell(self, step, scriptName):
             stepEnv["PATH"] = ":".join(step.getPaths() + step.getSandbox().getPaths())
         stepEnv["LD_LIBRARY_PATH"] = ":".join(step.getLibraryPaths())
         stepEnv["BOB_CWD"] = step.getExecPath()
+        stepEnv["BOB_STEP_NAME"] = "/".join(self.__currentPackage.getStack())
```

Comment:
Referencing ``self.__currentPackage`` looks wrong, ``step.getPackage()`` is more reasonable. In any case the variable should be named 'BOB_PACKAGE_PATH' because that's what it is in the end. In BOB_STEP_NAME I would expect something like "checkout", "build" and "package".

I have some mixed feelings though because a common package might be built only once but is actually found under several package paths.

BTW, the Jenkins implementation is missing too.